modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -37,8 +37,8 @@ def submit
       end
 
       def submit_supporting_documents
-        if params[:form_id] == '40-0247'
-          attachment = PersistentAttachments::MilitaryRecords.new(form_id: '40-0247')
+        if params[:form_id] == '40-0247' || params[:form_id] == '10-10D'
```

Comment:
We might want to update this if future form_ids get added to something like
`if ['40-0247', '10-10D'].include?(params[:form_id])`